### ISSUE TITLE
Create system that can distribute alerts via a wide range of methods

### ISSUE THREAD

**Issue by jleach** [BCDevOps/developer-experience#1455]
## Description

Our tenant customers have asked that we provide an alternative method to Rocket Chat for distributing platform alerts.

In an alignment meeting on July 27th we discussed a few options:

1. Posting alerts to UptimeRobot status updates. Not possible due to limitations in the API. Uptime Robot do expect to extend the API but have no date in mind when this will be available.

2. We could build another "dashboard" but we're building quite a few and its just one more thing to maintain and one more thing people need to find and goto for information.

3. We discussed reviving the GitHub PR based system where where platform services or DXC-AS members can post alerts via a pull request. Once the PR is reviewed and merged the alert is picked up by a `bot` and disseminated by different means (rocket chat, email, etc).

We settled on a PoC for #3 above.

## Done When

- [ ] There exists a repository where alerts can be posted. Once the the PR is merged into master it gets distributed via the Rocket Chat #alerts challenge and the distribution list mechanics (mautic?)

**Comment by jleach** [comment]
@mitovskaol Does this one match the conversation in yesterdays meeting?

**Comment by mitovskaol** [comment]
@jleach Can we please add this to the DoD:

Once the the PR is merged into master , the Mautic API gets invoked and an email goes to a specific distribution list (it does not exist yet, I suggest making a copy of the Platform Services Update distribution list and call it Platform Services Technical Updates and then people can unsubscribe from it if they don't want to receive them).

Test Plan:
1. I create a PR in the https://github.com/BCDevOps/platform-services-status-page-notifications/pulls repo that DXC team currently uses for posting messages to #devops-alerts channel in RC
2. I merge the PR to master
3. The update is posted to #devops-alerts channel in RC
4. An email is sent through the Mautic app to the emails on the Platform Services Technical Updates distribution list.
